lib/zinnia.js

```diff
@@ -48,6 +54,7 @@ const updateAllSourceFiles = async () => {
   const modules = await Promise.all(
     Object
       .values(ZINNIA_MODULES)
+      .filter(({ module }) => module.includes(MODULE_FILTER))
```

Comment:
I find `include()` check too hacky and possibly confusing. One can specify `MODULE_FILTER=r` to run both Zinnia modules (voyageR, spaRk).

Why not to check for equality? `MODULE_FILTER ? module === MODULE_FILTER : true`

Reply:
I'll change it to strict equality 👍

Reply:
It just makes the implementation more complicated, because we need to check for the empty filter case too

Reply:
> It just makes the implementation more complicated, because we need to check for the empty filter case too

Yeah. Maybe you can extract a small helper function to keep this more DRY?

```js
.filter(({ module }) => matchesModuleFilter(MODULE_FILTER, module))
```

Reply:
done https://github.com/filecoin-station/core/pull/323/commits/9c9aee6ea34df95a93e49b286b3faf8f7410cdb5 🙏